sdk/servicebus/service-bus/src/serializers/ruleResourceSerializer.ts

```diff
@@ -364,9 +357,14 @@ function buildSqlParameter(value: RawSqlParameter): SqlParameter {
  * @param value
  */
 export function getRawSqlParameters(parameters: SqlParameter[] | undefined): any {
-  if (!Array.isArray(parameters)) {
+  if (parameters == undefined) {
     return undefined;
   }
+
+  if (!Array.isArray(parameters)) {
+    throw new TypeError(`SQL parameters must be an array of SqlParameter instances or undefined`);
```

Comment:
`instance` implies classes, e.g. user would expect to have to pass `new SqlParameter`. However, SqlParameter is just an object type. So I suggest replacing `instances` with `objects` to make this more clear.

```suggestion
    throw new TypeError(`SQL parameters must be an array of SqlParameter objects or undefined`);
```

Reply:
Updated